File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height:", end="")
        h = cs50.get_int()
        if h>=0 and h<=23:
            break
    
    printpyramid(h)
    
def printpyramid(h):
    row=0
    for row in range (h):
        for space in range (h-row-1):
            print(" ", end="")
        for hex1 in range (row+1):
            print("#", end="")
        print("  ", end="")
        for hex2 in range (row+1):
            print("#", end="")
        print()
        

if __name__ == "__main__":
    main()
    
        

File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void printpyramid(int);

int main(void)
{
    int h;
    do
    {
        printf("Height:");
        h = get_int();
    }    
    while (h<0 || h>23);
    
    printpyramid(h);
}

    void printpyramid(int h)
    {
        int r;
        for (r=0; r<h; r++)
        {
            int s=0;
            while (s<(h-r-1))
            {
                printf(" ");
                s++;
            }
    
            int b=0;
            while (b<r+1)
            {
                printf("#");
                b++;
            }
            {
                printf("  ");
            }
            
            int bs=0;
            while (bs<r+1)
            {
                printf("#");
                bs++;
            }
            
            printf("\n");
        }
    }
